### ISSUE TITLE
2.4.6 - Describe topic or purpose IN or OUT of context?

### ISSUE THREAD

**Issue by anevins12** [w3c/wcag#3291]
Hello!

What is the normative stance on whether 2.4.6 requires labels to describe topic or purpose out of context?

**Comment by awkawk** [comment]
I'd say no they do not need to make sense out of context.

For example, in the WCAG 2.1 specification every note under a Success Criterion is marked up as an H5 heading, and to understand that you need to know about the context.

**Comment by anevins12** [comment]
Thanks Andrew. What do you think about this part of the Understanding doc? Should it be removed?

> This Success Criterion helps people who use screen readers by ensuring that labels and headings are meaningful when read out of context, for example, in a Table of Contents, or when jumping from heading to heading within a page.

For many moons, this was the reason I failed undescriptive control labels (which matched the accName.) I was wrong on that.

**Comment by patrickhlauke** [comment]
Yes, that seems unfortunate wording in the understanding

**Comment by patrickhlauke** [comment]
PR here https://github.com/w3c/wcag/pull/3295

**Comment by scottaohara** [comment]
I'm re-opening this since issues should be closed once the work is merged, not before... as otherwise it could lead people to think the problem was solved and that maybe the PR is unnecessary.
